framework/platform/bpl/uci/db/bpl_db.cpp

```diff
@@ -0,0 +1,75 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include "bpl/bpl_db.h"
+
+#include <mapf/common/logger.h>
+
+//////////////////////////////////////////////////////////////////////////////
+/////////////////////////////// Implementation ///////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+namespace beerocks {
+namespace bpl {
+bool db_add_param(const std::string &path, const std::string &object, const std::string &val)
+{
+    LOG(TRACE) << __func__ << " - NOT IMPLEMENTED!";
+    return false;
+}
+
+bool db_add_param(const std::string &path, const std::string &object, const int &val)
+{
+    LOG(TRACE) << __func__ << " - NOT IMPLEMENTED!";
+    return false;
+}
+
+bool db_set_param(const std::string &path, const std::string &object, const std::string &val)
+{
+    LOG(TRACE) << __func__ << " - NOT IMPLEMENTED!";
+    return false;
+}
+
+bool db_set_param(const std::string &path, const std::string &object, const int &val)
```

Comment:
Changed to use an unordered_map<string, string>, therefore using a reference is better since we're working with a container and do not want to copy the whole thing